docs/reference/api/networking.md

```diff
@@ -0,0 +1,286 @@
+| ------ | ----------------------- |
+| GET    | `/api/v1/networking/data-networks` |
+
+### Parameters
+
+None
+
+### Sample Response
+
+```json
+{
+    "result": [
+        {
+            "name": "internet",
+            "ip-pool": "172.250.0.0/24",
+            "dns": "8.8.8.8",
+            "mtu": 1460,
+            "status": {
+                "sessions": 0
+            }
+        }
+    ]
+}
+```
+
+## Create a Data Network
+
+This path creates a new Data Network.
+
+| Method | Path                    |
+| ------ | ----------------------- |
+| POST   | `/api/v1/networking/data-networks` |
+
+### Parameters
+
+- `name` (string): The Name of the Data Network (dnn)
+- `ip-pool` (string): The IP pool of the data network in CIDR notation. Example: `172.250.0.0/24`.
+- `dns` (string): The IP address of the DNS server of the data network. Example: `8.8.8.8`.
+- `mtu` (integer): The MTU of the data network. Must be an integer between 0 and 65535.
+
+### Sample Response
+
+```json
+{
+    "result": {
+        "message": "Data Network created successfully"
+    }
+}
+```
+
+## Update a Data Network
+
+This path updates an existing data network.
+
+| Method | Path                           |
+| ------ | ------------------------------ |
+| PUT    | `/api/v1/networking/data-networks/{name}` |
+
+### Parameters
+
+- `ip-pool` (string): The IP pool of the data network in CIDR notation. Example: `172.250.0.0/24`.
+- `dns` (string): The IP address of the DNS server of the data network. Example: `8.8.8.8`.
+- `mtu` (integer): The MTU of the data network. Must be an integer between 0 and 65535.
+
+### Sample Response
+
+```json
+{
+    "result": {
+        "message": "Data Network updated successfully"
+    }
+}
+```
+
+## Get a Data Network
+
+This path returns the details of a specific data network.
+
+| Method | Path                           |
+| ------ | ------------------------------ |
+| GET    | `/api/v1/networking/data-networks/{name}` |
+
+### Parameters
+
+None
+
+### Sample Response
+
+```json
+{
+    "result": {
+        "name": "internet",
+        "ip-pool": "0.0.0.0/24",
+        "dns": "8.8.8.8",
+        "mtu": 1460,
+        "status": {
+            "sessions": 0
+        }
+    }
+}
+```
+
+## Delete a Data Network
+
+This path deletes a data network from Ella Core.
+
+| Method | Path                           |
+| ------ | ------------------------------ |
+| DELETE | `/api/v1/networking/data-networks/{name}` |
+
+### Parameters
+
+None
+
+### Sample Response
+
+```json
+{
+    "result": {
+        "message": "Data Network deleted successfully"
+    }
+}
+```
+
+
+# Routes
+
+## List Routes
+
+This path returns the list of routes.
+
+
+| Method | Path             |
+| ------ | ---------------- |
+| GET    | `/api/v1/networking/routes` |
+
+### Parameters
+
+None
+
+### Sample Response
+
+```json
+{
+    "result": [
+        {
+            "id": 1,
+            "destination": "0.0.0.0/0",
+            "gateway": "66.66.66.6",
```

Comment:
This is very minor, and does not need to be changed right away, but there are IP ranges that are reserved for documentation that could be used instead of using IPs that are owner by some companies. This one for example is owned by Charter. I only put this comment here, but it would apply to the entirety of the documentation. [RFC5737](https://www.rfc-editor.org/rfc/rfc5737)

Reply:
Ok I moved it to `203.0.113.0/24`. In our tutorials we still use some addresses from reserved subnets (ex. `33.33.33.0/24`. I haven't changed that here since it would require a bit more effort (updating diagrams, `ip a` results, etc.) but it would be easy to do.